packages/fuels/test/utils/mockAutoStartFuelCore.ts

```diff
@@ -16,7 +15,7 @@ export const mockStartFuelCore = (): {
     port: 4000,
     providerUrl: `http://127.0.0.1:4000/graphql`,
     killChildProcess,
-    chainConfig: '/some/path/chainConfig.json',
+    chainConfigPath: '/some/path/chainConfig.json',
```

Comment:
We could declare this variable with a `FuelCoreNode` type annotation for increased clarity.

```ts
const fuelCore: FuelCoreNode = {
```

Also, since the original method is `async`, you should return a `Promise.resolve()`:

```ts
const autoStartFuelCore = vi
    .spyOn(autoStartFuelCoreMod, 'autoStartFuelCore')
    .mockResolvedValue(Promise.resolve(fuelCore));
```

Reply:
@arboleya I think it already takes into account the fact that the original method is `async` and _resolves_ the Promise with `fuelCore`. I tried adding in your suggestion and it gave me a type error saying the same.